crates/client/src/client/schema/tx.rs

```diff
@@ -178,7 +178,7 @@ pub struct SubmittedStatus {
 #[cynic(schema_path = "./assets/schema.sdl")]
 pub struct SuccessStatus {
     pub transaction_id: TransactionId,
-    pub block: BlockIdFragment,
+    pub block: BlockHeightFragment,
```

Comment:
It is named `block` since from the GraphQL perspective you receive the block=) The `block_height` is ID, but it could be `BlockId` as well(both works)

Reply:
Agreed - since this could return a full block, we must use a fragment query here to only return a partial view of the object.